app/assets/javascripts/student_profile/profile_chart_settings.jsx

```diff
@@ -77,11 +77,16 @@
     },
     tooltip: {
        formatter: function () {
-        return  Highcharts.dateFormat('%A, %b %e, %Y', new Date(this.x)) +
-                '<br>Percentile Rank:<b> ' +
-                  this.y +
-                '</b><br>Grade Level Equivalent: <b>' +
-                  this.points[0].point.gradeLevelEquivalent + '</b>';
+        var date = Highcharts.dateFormat('%A, %b %e, %Y', new Date(this.x))
+        var percentileRank = this.y
+        var gradeLevelEquivalent = this.points[0].point.gradeLevelEquivalent
+
+        if (gradeLevelEquivalent === 0 || null) {
```

Comment:
`null` is always going to evaluate as falsey, I think we want something like this instead: 

```
if (gradeLevelEquivalent === 0 || gradeLevelEquivalent === null) . . . 
```